src/utils/selectedCellUtils.ts

```diff
@@ -51,9 +51,8 @@ export function getNextSelectedCellPosition<R, SR>({
   isGroupRow
 }: GetNextSelectedCellPositionOpts<R, SR>): Position {
   const rowsCount = rows.length;
-  let position = nextPosition;
 
-  const setColSpan = (moveRight: boolean) => {
+  const setColSpan = (position: Position, moveRight: boolean): Position => {
```

Comment:
Does the name `setColSpan` make sense anymore?

I wonder if we even need a position object? Could we have something like this?
```ts
let { idx, rowIdx } = nextPosition;
```
Then we update `idx`/`rowIdx`, and at the end
```ts
return { idx, rowIdx };
```

Reply:
I will check